ci/cron/wednesday.yml

```diff
@@ -95,51 +81,10 @@ jobs:
           echo "$next" >> $tmp
           mv $tmp release/rotation
       }
-      release_message() {
-          local handler=$1
-          local start=$2
-          local commit_log
-          commit_log=$(git log --pretty=oneline $start..HEAD)
-          local changelog
-          changelog=$(./unreleased.sh $start..HEAD)
-          # Keep this line in sync with RELEASE_PR_NOTIF in azure-pipelines.yml
-          echo "This PR has been created by a script, which is not very smart"
-          echo "and does not have all the context. Please do double-check that"
-          echo "the version prefix is correct before merging."
-          echo ""
-          echo "@${handler} is in charge of this release."
-          echo ""
-          echo "Commit log:"
-          echo "\`\`\`"
-          echo "$commit_log"
-          echo "\`\`\`"
-          echo "Changelog:"
-          echo "\`\`\`"
-          echo "$changelog"
-          echo "\`\`\`"
-      }
```

Comment:
It was nice to have this summary done for you to present what's new after doing the release testing. Should we document on the release instructions how to generate this information?

Reply:
I'm not convinced the commit log added much, and IIRC the changelog was already documented, but I'll double-check.